sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/CbsAuthorizationType.java

```diff
@@ -8,7 +8,7 @@
 /**
  * An enumeration of supported authorization methods with the {@link ClaimsBasedSecurityNode}.
  */
-public enum CBSAuthorizationType {
+public enum CbsAuthorizationType {
```

Comment:
Do we need to expand CBS here just like we have done for other types?

Reply:
CbsAuthorizationType is used in the ClaimsBasedSecurityNode class, I thought it would be overkill.